vscode/src/services/SecretStorageProvider.ts

```diff
@@ -26,9 +27,18 @@ export interface SecretStorage {
 }
 
 export class VSCodeSecretStorage implements SecretStorage {
-    constructor(private secretStorage: vscode.SecretStorage) {}
+    private fsPath: string | null = null
+    constructor(private secretStorage: vscode.SecretStorage) {
+        const config = vscode.workspace.getConfiguration('cody')
+        // For user that does not have secret storage implemented in their sever
+        this.fsPath = config.get('experimental.LocalTokenPath') || null
```

Comment:
```suggestion
        this.fsPath = config.get('experimental.localTokenPath') || null
```

I think none of our settings start with a capitalized name or maybe I remember incorrectly 😅